profiles/base.yaml

```diff
@@ -471,6 +471,9 @@ priority_skills_lower_limit: 6
 # define one or multiple hunting zones (see base-hunting in scripts/data) and the behaviors for those hunts.
 # https://github.com/rpherbig/dr-scripts/wiki/YAML-Basics:-Part-2---Creating-Your-Own-YAML
 hunting_info:
+# Toggle to cause hunting buddy to stop a hunt short when the burgle
+# timer hits.
+stop_to_burgle: false
```

Comment:
Since this is only used by hunting_buddy, we could make it a part of `hunting_info` like we do with `boxes` and `stop_on` and `stop_on_low`, etc.

Reply:
That's reasonable. I originally coded it just for myself, and didn't give the setting name/location too much thought. :)

Reply:
Ahhh. I remember now why I did it this way. I have this setting in my include-main file, which all my characters include in their yamls. One file change, versus 16 (since all my characters have more than one hunting-buddy config file.) This makes me a little conflicted on the notion of putting this in hunting_info, tbh.

Reply:
How would you feel about having both options? I'd like to keep my universal setting, but that defaults to false, and I can add a toggle to `hunting_info` as well.

Reply:
Have a look at the current iteration? I've kept my global toggle, but added a per-hunt toggle as well.